scripts/webidl/convert.js

```diff
@@ -30,6 +30,7 @@ addDir("../../lib/jsdom/living/mutation-observer");
 addDir("../../lib/jsdom/living/navigator");
 addDir("../../lib/jsdom/living/nodes");
 addDir("../../lib/jsdom/living/svg");
+addDir("../../lib/jsdom/living/geometry");
```

Comment:
I see you've added quite a lot of tests, which makes me a bit nervous as they seem to contain a whole bunch of numbers. Have you checked if the [existing tests](https://github.com/web-platform-tests/wpt/tree/master/css/geometry) in WPT already cover these cases? If so, then let's just enable those tests by modifying https://github.com/jsdom/jsdom/blob/master/test/web-platform-tests/to-run.yaml.

If there is additional coverage provided by this PR, please put the tests into `css/geometry` in order to be consistent with upstream. Additionally, make sure to run these tests in the browser to make sure they're good.

Reply:
No I've not tested anything